model/model.go

```diff
@@ -14,18 +14,18 @@ import (
 
 // RouterConfig is the primary type used to encapsulate all router configuration.
 type RouterConfig struct {
-	WorkerProcesses          string      `json:"workerProcesses"`
-	MaxWorkerConnections     int         `json:"maxWorkerConnections"`
-	DefaultTimeout           int         `json:"defaultTimeout"`
-	ServerNameHashMaxSize    int         `json:"serverNameHashMaxSize"`
-	ServerNameHashBucketSize int         `json:"serverNameHashBucketSize"`
-	GzipConfig               *GzipConfig `json:"gzipConfig"`
-	BodySize                 int         `json:"bodySize"`
-	ProxyRealIPCIDR          string      `json:"proxyRealIpCidr"`
-	ErrorLogLevel            string      `json:"errorLogLevel"`
-	Domain                   string      `json:"domain"`
-	UseProxyProtocol         bool        `json:"useProxyProtocol"`
-	EnforceWhitelists        bool        `json:"enforceWhitelists"`
+	WorkerProcesses          string      `router:"workerProcesses"`
```

Comment:
Can you make this tag `deis-router` instead of just `router`? My hope is that it'll reduce the probability for collisions later